.github/workflows/ci.yml

```diff
@@ -89,9 +91,11 @@ jobs:
           wget -q https://raw.githubusercontent.com/nf-core/test-datasets/cutandrun/reference/genomes/e_coli_U00096_3.fa.gz
           wget -q https://raw.githubusercontent.com/nf-core/test-datasets/cutandrun/reference/genomes/e_coli_U00096_3.tar.gz
 
-      - name: Run pipeline with test data
-        run: |
-          nextflow run ${GITHUB_WORKSPACE} -profile docker,test_full_small_local_zip
+      - name: Run Pipeline
+        uses: Wandalen/wretry.action@v1.0.11
```

Comment:
What does this one do? It is not part of the template, is it?

Reply:
Hey, I get transient failures on tests that I cant control so I included an auto-retry module so that they always pass.

Reply:
nice, I have had the same problem and just been rerunning that test manually, ever since GHA allows to only rerun failed jobs. this is obviously much nicer